website/docs/concepts/router.md

```diff
@@ -135,6 +135,42 @@ fn app() -> Html {
 }
 ```
 
+### Add Parameters
+
+It is also possible to extract information from a route.
+
+```rust
+# use yew_router::prelude::*;
+#[derive(Clone, Routable, PartialEq)]
+enum Route {
+    #[at("/")]
+    Home,
+    #[at("/post/:id")]
+    Post { id: String },
+    // ...
+}
+```
+
+You can then access the post's id inside `<Switch />` and forward it to the appropriate component via properties.
+
+```rust,ignore
```

Comment:
These are not highlighting rust syntax, I wonder whats up
![image](https://user-images.githubusercontent.com/32368314/142759754-2e0f2cba-2eb3-43fa-93ac-0c6f8fe1656d.png)

Reply:
Need to add a space after `rust` :) because Docusaurus' syntax highlight plugin gets confused by the trailing comma 🙃

Reply:
```suggestion
```rust ,ignore
```